### ISSUE TITLE
Results are weird

### ISSUE THREAD

**Issue by ixtiyoruz** [ialhashim/DenseDepth#48]
![image](https://user-images.githubusercontent.com/24115387/62844020-37c44900-bcf9-11e9-830a-c37ded77425a.png)
I have used it with Yolo, to measure the distance but the results are wrong. Any help would be appreciated?
```

    last_time = time.time()
    ret, frame = cap.read()
    detections = darknet_video.detect_from_image(frame)
    
    original_height, original_width , _ = np.shape(frame)
    detections = darknet_video.resize_detections(detections,rgb_height,  rgb_width)
    
    frame = cv2.resize(frame, (rgb_width, rgb_height ), interpolation=cv2.INTER_CUBIC)
    # Compute results
    outputs = predict(model, np.expand_dims(frame/256,0)) * 80 
    
    disp_resized = cv2.resize(outputs[0], (rgb_width ,rgb_height ), interpolation=cv2.INTER_CUBIC) 

    #pos = posFromDepth(disp_resized)
    #print(np.shape(pos))
    print(np.shape(disp_resized), np.shape(frame))
    detections = get_min_disp_dets(disp_resized, detections,rgb_height , rgb_width )
    colormapped_im = darknet_video.cvDrawBoxes_with_distance(detections, frame)
    #print(1./(time.time() - last_time))
    cv2.imshow('image', colormapped_im)
    cv2.imshow('image1', frame)
    if cv2.waitKey(1) & 0xFF == ord('q'):
        break


```

**Comment by ialhashim** [comment]
Do you have the ground truth? if not then its hard to evaluate how wrong the results are. 

In any case this seems like a test from a dataset that is not too similar to KITTI (this is relevant if you are using the model pretrained on KITTI). It is important to train models on a similar setup as the expected test set (e.g. same camera, subject matter). Otherwise, the performance will be very low when the model is predicting on cases outside the range of images it is trained on (in that case a refinement step can help greatly, provided that the depth information is available).

Keep in mind, most supervised methods out there (including ours) do not generalize well to unseen data. So do not expect good performance without having large (and relevant) training data.

(PS: you can try to modify the demo I added to show the point cloud that might help in debugging.)

**Comment by ixtiyoruz** [comment]
Thank you for the answer. Yes I dont have ground truth.
